# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1256855974 14400
#      Thu Oct 29 18:39:34 2009 -0400
# Node ID 63d554a54da6d86d9d0073b2a6ddec96e3b4a8cb
# Parent  8e277dcf1223532409b9c23cae4f2ba416dca59c
configure.ac: fix logic in previous change

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1581,17 +1581,17 @@ if $SHARED_LIBS || $ENABLE_DYNAMIC_LINKI
   save_LIBS="$LIBS"
   use_mach_dyld=false
   AC_ARG_ENABLE(mach-dyld,
     [AS_HELP_STRING([--enable-mach-dyld], [enable older Mach-O dyld interface])],
       [if test "x$enableval" = xno; then
          use_mach_dyld=false;
        else
          use_mach_dyld=true;
-       fi], [use_mach_dyld=true])
+       fi], [])
   if $use_mach_dyld; then
     AC_CHECK_HEADER(mach-o/dyld.h)
     if test "$ac_cv_header_mach_o_dyld_h" = yes; then
       dyld_api=true
     fi
   elif test "$dyld_api" = false; then
     AC_CHECK_LIB(dld, shl_load, [DL_LIBS=-ldld; LIBS="$LIBS $DL_LIBS"])
     AC_CHECK_FUNCS(shl_load shl_findsym)
